style.css

```diff
@@ -1381,6 +1383,226 @@
+    line-height: 52px;
+
+    /* or 130% */
+
+    text-align: center;
+
+    /* Neutrals/N0 */
+
+    color: #fff;
+  }
+
+  .contact-par {
+    width: 327px;
+    height: 140px;
+    left: 24px;
+    top: 4802px;
+    margin-top: 12px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+
+    /* or 140% */
+
+    text-align: center;
+
+    /* Primary/B50 */
+
+    color: #ebebff;
+  }
+
+  .contact-form {
+    width: 327px;
+  }
+
+  .con-f-list {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-wrap: nowrap;
+    padding: 0;
+  }
+
+  .con-f-list li + li {
+    margin-top: 18px;
+  }
+
+  .name,
+  .email,
+  .feedback {
+    /* Mono/White */
+
+    background: #fff;
+
+    /* Base/100 */
+
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    width: 327px;
+  }
+
+  input {
+    height: 48px;
+  }
+
+  .feedback {
+    height: 170px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+
+    /* or 160% */
+
+    /* Neutrals/N60 */
+
+    color: #b3bac5;
+  }
+
+  .name {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+
+    /* or 118% */
+
+    display: flex;
+    align-items: center;
+
+    /* Neutral/N800 */
+
+    color: #172b4d;
+  }
+
+  .email {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+
+    /* or 118% */
+
+    display: flex;
+    align-items: center;
+
+    /* Neutral/N800 */
+
+    color: #172b4d;
+  }
+
+  .con-f-list > li > button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+
+    /* Neutrals/N0 */
+
+    background: #fff;
+    border-radius: 8px;
+
+    /* Button /Normal */
+
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+
+    /* identical to box height, or 141% */
+
+    text-align: center;
+    letter-spacing: 0.03em;
+
+    /* Primary/B400 */
+
+    color: #6070ff;
+    border-color: #fff;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  input::placeholder {
+    padding-left: 10px;
+  }
+
+  textarea::placeholder {
+    padding-left: 10px;
+    padding-top: 10px;
+  }
+
+  input:focus {
+    padding-left: 10px;
+  }
+
+  textarea:focus {
+    padding-left: 10px;
+    padding-top: 10px;
+  }
+
+  .form-button:hover {
+    background: #6070ff;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+    color: #fff;
+  }
+
+  .form-button:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .form-button:disabled {
+    border: 1px solid #c1c7d0;
+    color: #5e6c84;
+  }
 }
 
 @media only screen and (min-width: 375px) {
```

Comment:
- Kindly I suggest you use the  appropriate screen size  I think that is very small and your page is not responsive in the mobile version I suggest you use for example min-width:768px

Reply:
Hello, In the Figma design, the mobile design requirements are at a width of 375px. My previous code reviewers also recommended using the design width.  So should still change it?

Reply:
@MaganaAsiati

Reply:
Hello @gaks1 , That's right. the mobile version should be `375px`. I'll review your code and end my suggestions. Please check them below